runtime/near-vm-logic/src/gas_counter.rs

```diff
@@ -22,6 +22,8 @@ pub struct GasCounter {
     prepaid_gas: Gas,
     is_view: bool,
     ext_costs_config: ExtCostsConfig,
+    /// Where to store profile data, if needed.
```

Comment:
If I understood you correctly @olonho you said that it creates a negligible overhead if so then we can try to drop `costs_counting` flag once we get rid of the thread local static variable.

Reply:
Not sure if we could, see comment above. It's used in https://github.com/nearprotocol/nearcore/blob/f024a28d9b0f1b2238f04d288ddebc8ac12ff4d1/runtime/runtime-params-estimator/src/stats.rs#L29.